_posts/2019-10-24-writing-a-compatible-date-input.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Writing a Browser Compatible Date Input"
+date: 2019-10-24 13:00:00
+tags:
+- JavaScript
+- Vue
+- HTML
+categories:
+- JavaScript
+twitter_text: "Writing a Browswer Compatible Date Input"
```

Comment:
```suggestion
twitter_text: "Writing a Browser Compatible Date Input"
```

Reply:
How about `Writing a Bowser Compatible Date Input`? :smile:

Reply:
Save that one for April 1.